From 3b9f4786a78e173f13e912208ce0903520bf8a58 Mon Sep 17 00:00:00 2001
From: Ivan Udovichenko <iudovichenko@mirantis.com>
Date: Sat, 19 Sep 2015 01:33:54 +0300
Subject: [PATCH] Remove django-discover-runner from test_require list in
 setup.py

Bump up version of Django.
As Django >= 1.6 already includes discover-runner.
---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index c09af07..a7a358c 100644
--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,7 @@ def read(fname):
 
 
 install_requires = [
-    'Django>=1.4',
+    'Django>=1.6',
     'pyScss>=1.3.4',
 ]
 
@@ -22,7 +22,6 @@ except ImportError:
 tests_require = [
     'Pillow',
     'django-compressor>=1.3',
-    'django-discover-runner',
     'mock',
 ]
 
-- 
2.1.4

